pkg/liqoctl/info/peer/offloading.go

```diff
@@ -0,0 +1,163 @@
+		} else {
+			oc.collectVirtualNodes(virtualNodeList.Items, &offloadingState)
+		}
+		oc.data[clusterID] = offloadingState
+	}
+}
+
+// FormatForClusterID returns the collected data for the specified clusterID using a user friendly output.
+func (oc *OffloadingChecker) FormatForClusterID(clusterID liqov1beta1.ClusterID, options info.Options) string {
+	if data, ok := oc.data[clusterID]; ok {
+		main := output.NewRootSection()
+		main.AddEntry("Status", common.FormatStatus(data.Status))
+		if data.Status != common.ModuleDisabled {
+			// Show alerts if any
+			if len(data.Alerts) > 0 {
+				main.AddEntryWarning("Alerts", data.Alerts...)
+			}
+
+			// Show virtual nodes
+			nodesSection := main.AddSection("Virtual nodes")
+			for i := range data.VirtualNodes {
+				vNode := &data.VirtualNodes[i]
+				currNodeSection := nodesSection.AddSection(vNode.Name)
+				currNodeSection.AddEntry("Status", common.FormatStatus(vNode.Status))
+				currNodeSection.AddEntry("Secret", vNode.Secret)
+				currNodeSection.AddEntry("Resource slice", vNode.ResourceSlice)
+				resourcesSection := currNodeSection.AddSection("Resources")
+				for resource, quantity := range vNode.Resources {
+					resourcesSection.AddEntry(string(resource), quantity.String())
+				}
+			}
+		}
+		return main.SprintForBox(options.Printer)
+	}
+	return ""
+}
+
+// GetDataByClusterID returns the data collected by the checker for the cluster with the give ClusterID.
+func (oc *OffloadingChecker) GetDataByClusterID(clusterID liqov1beta1.ClusterID) (interface{}, error) {
+	if res, ok := oc.data[clusterID]; ok {
+		return res, nil
+	}
+	return nil, fmt.Errorf("no data collected for cluster %q", clusterID)
+}
+
+// GetID returns the id of the section collected by the checker.
+func (oc *OffloadingChecker) GetID() string {
+	return "offloading"
+}
+
+// GetTitle returns the title of the section collected by the checker.
+func (oc *OffloadingChecker) GetTitle() string {
+	return "Offloading"
+}
+
+// collectStatusInfo collects the info about the status of the offloading module.
+func (oc *OffloadingChecker) collectStatusInfo(clusterID liqov1beta1.ClusterID,
+	clusterInfo map[liqov1beta1.ClusterID]*liqov1beta1.ForeignCluster, offloadingState *Offloading) {
+	cluster := clusterInfo[clusterID]
+	offloadingState.Status, offloadingState.Alerts = common.CheckModuleStatusAndAlerts(cluster.Status.Modules.Offloading)
+}
+
+// collectVirtualNodes collects the data from the list of VirtualNode CRs.
+func (oc *OffloadingChecker) collectVirtualNodes(virtualNodes []offloadingv1beta1.VirtualNode, offloadingState *Offloading) {
+	offloadingState.VirtualNodes = []VirtualNodeStatus{}
+	for i := range virtualNodes {
+		vNode := &virtualNodes[i]
+
+		// Check the status of the virtual node
+		status := common.ModuleUnhealthy
+		for _, condition := range vNode.Status.Conditions {
+			if condition.Status == offloadingv1beta1.RunningConditionStatusType {
+				status = common.ModuleHealthy
+			}
+		}
```

Comment:
Also here i would show Healthy only if all conditions are Healthy.
It could be necessary to consider also `None` status as healthy, as for example when you create a VirtualNode with the k8s Node disabled.